RealmSwift/SwiftUI.swift

```diff
@@ -21,13 +21,31 @@ import SwiftUI
 import Combine
 import Realm
 
-private func resolve<T: ThreadConfined>(_ frozen: T) -> (Realm, T)? {
-    guard let config = frozen.realm?.configuration,
-          let realm = try? Realm(configuration: config),
-          let resolved = realm.thaw(frozen) else {
-        return nil
-    }
-    return (realm, resolved)
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+private func createBinding<T: ThreadConfined, V>(_ getter: @escaping () -> T,
+                                                 forKeyPath keyPath: ReferenceWritableKeyPath<T, V>) -> Binding<V> {
+    let lastValue = getter()[keyPath: keyPath]
+    return Binding(get: {
+        var parent = getter()
+        guard !parent.isInvalidated else {
+            return lastValue
+        }
+        if parent.isFrozen {
+            parent = try! Realm(configuration: parent.realm!.configuration).thaw(parent)!
```

Comment:
Can't this just be `parent.realm!.thaw(parent)`?

The force-unwrap is unsafe and this'll crash if the object was deleted after the version which the parent was frozen at.